src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import java.util.*;
+
+public class UtilsTest {
+	
+	private Utils utils = new Utils();
+	
+	@Test public void isNullTest() {
```

Comment:
Note the naming convention for test methods (the first part is feature/method under test), e.g. `isAnyNull_emptyList_falseReturned`